docs/REST-API-Reference.md

```diff
@@ -196,6 +196,7 @@ Name | Type
 `parent_uuid` | String
 `parent_path` | String
 `parent_name` | Method
+`parent_state` | Method
```

Comment:
What does "Method" even mean? How can a REST API return a "method"?

Reply:
Shouldn't this just be a "string" type (and similarly for parent_name)?

Reply:
@epicfaace This was auto-generated. I think it signifies that some logic runs in the schema to get this value (specifically `get_parent_name()` and `get_parent_state()`.

Reply:
String probably makes more sense in terms of the docs tho....not sure how these are generated